airflow/api/client/json_client.py

```diff
@@ -24,9 +24,24 @@
 
 
 class Client(api_client.Client):
-    """Json API client implementation."""
+    """Json API client implementation.
 
-    def _request(self, url, method="GET", json=None):
+    This client is used to interact with a Json API server and perform various actions
+    such as triggering DAG runs,deleting DAGs, interacting with pools, and getting lineage information.
+
+    :param _api_base_url: The base URL for the Json API server.
+    :param _session: A session object to use for making HTTP requests.
```

Comment:
```suggestion
    :param api_base_url: The base URL for the Json API server.
    :param session: A session object to use for making HTTP requests.
```

As far as I can tell, the parameters don’t have an underscore prefix?